R/join_ald_scenario.R

```diff
@@ -1,24 +1,14 @@
 #' Join a data-loanbook object to the ald and scenario
 #'
-#' `join_ald_scenario()` is a simple wrapper of several calls to
-#' `dplyr::join_*()`, forming the master dataset to be used in later steps of
-#' the analysis.
+#' @description
+#' `r lifecycle::badge("deprecated")`
 #'
-#' @param data A data frame like the output of
-#'   `r2dii.match::prioritize`.
-#' @param ald An asset level data frame like [r2dii.data::ald_demo].
-#' @param scenario A scenario data frame like [r2dii.data::scenario_demo_2020].
-#' @param region_isos A data frame like [r2dii.data::region_isos] (default).
-#' @param add_green_technologies Logical vector of length 1. `FALSE` defaults to
-#' outputting only technologies that are present in both `data` and `ald`. Set
-#' to `FALSE` to add rows of all possible green technologies (with 0
-#' production).
+#' This function was deprecated in favour of [join_abcd_scenario()]. See [here](https://2degreesinvesting.github.io/posts/2022-03-02-ald-becomes-abcd/)
+#' for more information.
 #'
-#' @return Returns a fully joined data frame, linking portfolio, ald and
-#'   scenario.
-#' @export
+#' @keywords internal
 #'
-#' @family utility functions
+#' @export
```

Comment:
I assume this new export is so that this function IS available while it's in this deprecated state?

Reply:
Exactly. The function should be available, just advocated against using.